apps/cowswap-frontend/src/modules/application/containers/AppMenu/styled.ts

```diff
@@ -0,0 +1,11 @@
+import { Media } from '@cowprotocol/ui'
+
+import styled from 'styled-components/macro'
+
+export const isMobileQuery = Media.upToMedium
+
+export const HideMobile = styled.div`
```

Comment:
I see HideMobile and isMobileQuery are duplicated in MenuBar/styled.ts. It makes sense to get rid of the duplication by extracting it to a common file.

Reply:
AppMenu and MenuBar have different `isMobile` implementation.

- AppMenu uses `Media.upToMedium`
- MenuBar uses `Media.upToLarge`

So we can't use the same `<HideMobile>` for both components.

And if you check any another `isMobile` definition throughout the whole project, it's a common logic: every component interprets "isMobile" depending on it's own logic.

That's why I decided to extract it in `styled.ts` file per component. It highly depends on component logic.